js/apps/account-ui/src/api/methods.ts

```diff
@@ -109,8 +109,14 @@ export async function getCredentials({ signal, context }: CallOptions) {
   return parseResponse<CredentialContainer[]>(response);
 }
 
-export async function getLinkedAccounts({ signal, context }: CallOptions) {
-  const response = await request("/linked-accounts", context, { signal });
+export async function getLinkedAccounts(
+  { signal, context }: CallOptions,
+  requestParams: Record<string, any>,
```

Comment:
Same here, let's avoid using `any` if the parameters can be defined explicitly in their own type.